[2020-06-10 09:57:16,107] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: generator.generate_file 2017-01-23T20:00:00+00:00 [queued]>
[2020-06-10 09:57:16,113] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: generator.generate_file 2017-01-23T20:00:00+00:00 [queued]>
[2020-06-10 09:57:16,113] {taskinstance.py:879} INFO - 
--------------------------------------------------------------------------------
[2020-06-10 09:57:16,113] {taskinstance.py:880} INFO - Starting attempt 1 of 2
[2020-06-10 09:57:16,113] {taskinstance.py:881} INFO - 
--------------------------------------------------------------------------------
[2020-06-10 09:57:16,276] {taskinstance.py:900} INFO - Executing <Task(PythonOperator): generate_file> on 2017-01-23T20:00:00+00:00
[2020-06-10 09:57:16,278] {standard_task_runner.py:53} INFO - Started process 12129 to run task
[2020-06-10 09:57:16,421] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: generator.generate_file 2017-01-23T20:00:00+00:00 [running]> strx
[2020-06-10 09:57:16,429] {logging_mixin.py:112} INFO - Generating  1591797436 ...
[2020-06-10 09:57:16,430] {logging_mixin.py:112} INFO - Finished.
[2020-06-10 09:57:18,432] {python_operator.py:114} INFO - Done. Returned value was: _
[2020-06-10 09:57:18,643] {taskinstance.py:1065} INFO - Marking task as SUCCESS.dag_id=generator, task_id=generate_file, execution_date=20170123T200000, start_date=20200610T135716, end_date=20200610T135718
[2020-06-10 09:57:25,977] {logging_mixin.py:112} INFO - [2020-06-10 09:57:25,976] {local_task_job.py:103} INFO - Task exited with return code 0
